src/fmu/sumo/uploader/_connection.py

```diff
@@ -35,34 +35,3 @@ def refresh(self):
     def _establish_connection(self):
         """Establish the connection with Sumo API, take user through 2FA."""
         return SumoClient(env=self.env, token=self.token)
-
-
-class SumoConnectionWithOutsideToken:
```

Comment:
From what I can tell we don't use this class in any of our repos.
Also, functionality seems to be pretty much exact same as `SumoConnection` (might have been different before?)

Reply:
Never seen it in use myself. So seems rigtht.